apps/cowswap-frontend/src/modules/permit/hooks/useCheckHasValidPendingPermit.ts

```diff
@@ -78,9 +92,10 @@ async function checkIsSingleCallDataAValidPermit(
   eip2162Utils: Eip2612PermitUtils,
   tokenAddress: string,
   tokenName: string,
-  callData: string
+  callData: string,
+  { version }: SupportedPermitInfo
 ): Promise<boolean | undefined> {
-  const params = { chainId, tokenName, tokenAddress, callData }
+  const params = { chainId, tokenName: fixTokenName(tokenName), tokenAddress, callData, version }
```

Comment:
We need the USDC name fix and the version for successfully recovering USDC signatures